plugins/janus_audiobridge.c

```diff
@@ -868,6 +872,9 @@ typedef struct janus_audiobridge_participant {
 	/* Opus stuff */
 	OpusEncoder *encoder;		/* Opus encoder instance */
 	OpusDecoder *decoder;		/* Opus decoder instance */
+	uint16_t expected_seq;      /* Expected sequence number */
+	uint16_t probation; 		/* Used to determine new ssrc validity */
+	uint32_t last_timestamp;	/* Last in seq timestamp */
```

Comment:
Same here: tabs, not spaces (definitely an issue for `expected_seq` at the moment).

Reply:
done